Formula/wireshark.rb

```diff
@@ -13,10 +13,9 @@ class Wireshark < Formula
     sha256 "54e3add2e9631aa400bca0f06a9952dc16d620a2db3a21f1bb450e94c3f8f493" => :mavericks
   end
 
-  option "with-gtk+3", "Build the wireshark command with gtk+3"
-  option "with-gtk+", "Build the wireshark command with gtk+"
-  option "with-qt", "Build the wireshark-qt command (can be used with or without either GTK option)"
-  option "with-qt5", "Build the wireshark-qt command with qt5 (can be used with or without either GTK option)"
+  option "with-qt", "Build the wireshark command with Qt5 (can be used with or without either GTK option)"
```

Comment:
It'll still be `qt5` with us, so you'll want this option name to remain `with-qt5`.

Can you also add a `deprecated_option` which will nudge existing `wireshark --with-qt` users over to `wireshark --with-qt5` more seamlessly next time they upgrade/reinstall than simply yanking the option. The syntax for that is:

```ruby
deprecated_option "with-qt" => "with-qt5"
```

That should be placed one empty line beneath the last option.